api/src/pcapi/scripts/payment/user_recredit.py

```diff
@@ -118,3 +132,5 @@ def recredit_underage_users() -> None:
 
         start_index += RECREDIT_BATCH_SIZE
     logger.info("Recredited %s users successfully", total_users_recredited)
+    if failed_users:
+        logger.error("Failed to recredit %s users: %s", len(failed_users), failed_users)
```

Comment:
pas redondant avec `logger.exception("Could not recredit user %s: %s", user.id, e)`

Reply:
Je pense que c'est pas si mal de garder les 2
`logger.exception("Could not recredit user %s: %s", user.id, e)` donne plus de détails
`logger.error("Failed to recredit %s users: %s", len(failed_users), failed_users)` sert de bilan à la fin

On peut en discuter si tu trouves auqnd même que c'est redondant